src/coreclr/vm/bundle.cpp

```diff
@@ -80,11 +102,27 @@ BundleFileLocation Bundle::Probe(const SString& path, bool pathIsBundleRelative)
             return loc;
         }
     }
-
+#endif // !TARGET_ANDROID
     INT64 fileSize = 0;
     INT64 compressedSize = 0;
+    bool assemblyFound = false;
+
+    if (m_probe != nullptr)
+    {
+        assemblyFound = m_probe(utf8Path, &loc.Offset, &fileSize, &compressedSize);
```

Comment:
This is a slight change in the existing behavior. Currently the existing probe is defined as returning bool, but the code ignored it, now it doesn't. Given that we don't really expose this probe as an external API it's probably OK, but we should make it a concious decision.